annotated_types/__init__.py

```diff
@@ -64,34 +78,51 @@
 # positional only
 # see https://peps.python.org/pep-0484/#positional-only-arguments
 
+_T_contra = TypeVar('_T_contra', contravariant=True)
 
-class SupportsGt(Protocol):
-    def __gt__(self: T, __other: T) -> bool:
+
+class SupportsGt(Protocol[_T_contra]):
+    def __gt__(self, __other: _T_contra) -> bool:
         ...
 
 
-class SupportsGe(Protocol):
-    def __ge__(self: T, __other: T) -> bool:
+class SupportsGe(Protocol[_T_contra]):
+    def __ge__(self, __other: _T_contra) -> bool:
```

Comment:
Any reason not to use native positional-only parameters (`/`) here?

Reply:
Nope I just keep forgetting that Python 3.7 is EOL so / is fair game now :)